src/components/views/Game.js

```diff
@@ -96,13 +97,16 @@ const Game = () => {
   }
 
   return (
+      <React.Fragment>
```

Comment:
What does React.Fragment do?

Reply:
Allows for passing more than 1 html object